config.toml

```diff
@@ -112,12 +112,26 @@ archived_version = false
 # The version number for the version of the docs represented in this doc set.
 # Used in the "version-banner" partial to display a version number for the
 # current doc set.
-version = "0.0"
+version = "v0.1.1"
 
 # A link to latest version of the docs. Used in the "version-banner" partial to
 # point people to the main doc site.
 url_latest_version = "https://cdevents.dev/docs"
 
+# Maintain history of release versions here
+# This section controls the content of the Releases menu
+[[params.versions]]
+  version = "latest"
+  url = "https://cdevents.dev/docs"
+
+[[params.versions]]
+  version = "v0.1.1"
+  url = "https://github.com/cdevents/spec/tree/v0.1.1"
```

Comment:
Perhaps we should point this to https://cdevents.dev/docs, and switch it to "https://github.com/cdevents/spec/tree/v0.1.1" when a more recent release becomes available?

Reply:
We'll also need to document the need to update this file when doing a release - we don't have yet a document that defines the release process for spec / sdks, we'll need to add it in there once available.

Reply:
The intention was to delete this instance once you are happy with the way the menu works. It’s just there to give more content at the moment.

I have documented the release process in https://github.com/cdevents/cdevents.dev/issues/15

Reply:
I have removed, temporarily. We will restore when 0.2.0 is published.